src/components/analyser/search/StockListOverview.tsx

```diff
@@ -46,7 +47,13 @@ const StockListOverview: React.FC<StockListOverviewProps> = ({
           />
         ) : (
           // TODO: implement card view
-          <div>cards</div>
+          <div>cards
```

Comment:
This does not seem to be done yet, "cards" label is still here

Reply:
done